content/en/security_platform/cloud_workload_security/workload_security_rules.md

```diff
@@ -15,23 +15,27 @@ further_reading:
 
 ## Overview
 
-With Cloud Workload Security (CWS) enabled, the Datadog Agent actively monitors system activity and evaluates it against a set of rules to detect suspicious behavior. Before learning how to create custom rules below, it is recommended to learn the basics of Cloud Workload Security rules and the concept of Agent expressions. Refer to the [Agent expression docs][1] for more information.
+With Cloud Workload Security (CWS) enabled, the Datadog Agent actively monitors system activity and evaluates it against a set of rules to detect suspicious behavior. 
 
-The full set of rules for the Agent is called a policy. Datadog provides several [out-of-the-box Cloud Workload Security rules][2] for your convenience in the default policy.
+When you upgrade the Datadog Agent, you receive [bundled CWS Agent rules][1]. You can also write your own custom Agent rules. This guide covers Agent rules and how to create them.
```

Comment:
When you upgrade the Datadog Agent, you receive bundled CWS Agent rules used in our [default Signal Rules ](https://docs-staging.datadoghq.com/sarina/cws-rule-split/security_platform/default_rules/#cat-workload-security). You can also write your own custom Agent rules. This guide covers Agent rules, how to create them, and how to use them to generate security signals.

Reply:
The URL is changed to filter by CWS rules